src/constants.ts

```diff
@@ -68,6 +68,27 @@ export const ALLOWED_FILETYPES_LABEL = Object.entries(MIMETYPE)
   .map((e) => (e[0] === 'ZIP' ? 'HTML (ZIP ARCHIVE)' : e[0]))
   .join(', ')
 
+export const FEED_MAP = {
```

Comment:
I was referring to this, I don't think it's needed

Reply:
I simplified it to just a list, I can't fully test it but I just pushed a few edits, I'll be able to properly test it tomorrow and merge then! Thanks again

Reply:
Thanks @melMass!